src/main/java/dev/nolij/zson/Zson.java

```diff
@@ -127,6 +129,60 @@ public static String escape(String string, char escapeQuotes) {
 		return result.toString();
 	}
 
+	public static Map<String, ZsonValue> obj2Map(Object object) {
+		Map<String, ZsonValue> map = Zson.object();
+		for (Field field : object.getClass().getDeclaredFields()) {
+			if(Modifier.isStatic(field.getModifiers())) continue;
```

Comment:
Please also exclude non-public fields, and add an annotation to exclude public non-final fields as well.